code/modules/reagents/chemistry/chem_wiki_render.dm

```diff
@@ -0,0 +1,199 @@
+//Generates a wikitable txt file for use with the wiki - does not support productless reactions at the moment
+/client/proc/generate_wikichem_list()
+	set category = "Debug"
```

Comment:
Indent with tabs, not spaces.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

Reply:
Are you sure it's spaces? VSC seems to think it's tabs and doesn't detect any changes when I use conver to indentation.

Reply:
I think it's just GitHub being shitty, actually.